test/linalg/rowvector.jl

```diff
@@ -268,3 +268,19 @@ end
 @testset "ambiguity between * methods with RowVectors and ConjRowVectors (#20971)" begin
     @test RowVector(ConjArray(ones(4))) * ones(4) == 4
 end
+
+@testset "setindex!/getindex" begin
+    v = [2,3,4]
```

Comment:
I have been trying to develop a model of when you do / do not inject spaces but haven't been successful yet ;).